notebooks/community/managed_notebooks/predictive_maintainance/Predictive_maintenance_usecase.ipynb

```diff
@@ -0,0 +1,1105 @@
+{
```

Comment:
<p>"Vertex AI" instead of"Vertex-AI"</p><p>"URI" instead of "uri"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>resolved</p><p> </p>